jaws-journey-deploy/orb.yml

```diff
@@ -539,6 +539,53 @@ jobs:
             - slack/notify:
                 <<: *slack-notification
 
+  gitops-deploy:
+    working_directory: ~/working
+    machine: true
+    parameters:
+      manifest_directory:
+        type: string
+        default: "manifests"
+      git_email:
+        type: string
+        default: "jaws.devsdl@ovoenergy.com"
+      git_name:
+        type: string
+        default: "Journeys CircleCI Orb"
+      ssh_key_fingerprint:
+        type: string
+      yq_version: 
+        type: string
+        default: "v4.6.2"
+      <<: *environment
+    description: Deploy to << parameters.environment >>
+    environment:
+      <<: *tf-mask
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - << parameters.ssh_key_fingerprint >>
+      - run:
+          name: "Install yq"
```

Comment:
could we move the commands into the command section like we do with other jobs?  Also maybe move the commands themselves into the scripts folder?  Mostly for consistency and so the yml file isnt HUGE (although its already pretty big